.github/workflows/awesome_workflow.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: requirements
         run: | 
           sudo apt -qq -y update
-          sudo apt -qq install clang-format
+          sudo apt -qq install clang-tidy-10
```

Comment:
checks are passing with less errors when used with this version. The default installs v6.0 which did not work out well in my tests